target/linux/ath79/dts/qca9531_joyit_jt-or750i.dts

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca953x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "joyit,jt-or750i", "qca,qca9531";
+	model = "Joy-IT JT-OR750i";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_red;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_red;
+		label-mac-device = &eth1;
```

Comment:
I've obtained the following assignment, so this should be &eth0?

```
MAC addresses:

flash   addr  DT     device (userspace)
0x0     *:9c  &eth0  eth1 (wan), label
0x6     *:9d  &eth1  eth0 (lan)
0x1002  *:9e  2.4g   radio1
0x5006  *:9f  5g     radio0
```

Please add the table to the commit message if you verified it ...

Reply:
You're right.